doc/apps/dgst.pod

```diff
@@ -14,21 +14,24 @@ B<openssl> B<dgst>
 [B<-binary>]
 [B<-r>]
 [B<-non-fips-allow>]
-[B<-out filename>]
-[B<-sign filename>]
-[B<-keyform arg>]
-[B<-passin arg>]
-[B<-verify filename>]
-[B<-prverify filename>]
-[B<-signature filename>]
-[B<-hmac key>]
+[B<-out> I<filename>]
+[B<-sign> I<filename>]
+[B<-keyform> I<arg>]
+[B<-passin> I<arg>]
+[B<-verify> I<filename>]
+[B<-prverify> I<filename>]
+[B<-signature> I<filename>]
+[B<-hmac> I<key>]
+[B<-mac> I<alg>]
+[B<-macopt> I<nm>B<:>I<v> ...]
+[B<-rand> I<files>]
 [B<-non-fips-allow>]
 [B<-fips-fingerprint>]
-[B<file...>]
+[I<file ...>]
```

Comment:
You mean the one in `B<openssl> I<digest> [...]`? Because there, the ellipsis has a broader meaning: instead of meaning "the last item, optionally repeated", it means "other stuff", because it's standing in for any of the other options used by the `B<openssl dgst>` form. In the places where I've italicized `...`, it's because it's repeating a replacement-text item which itself is completely italicized (that is, repeating the last replacement-text token); in this case, it stands in for a mix of other elements.

Since this `...` doesn't really mean "repetition", it should probably be something else entirely, like maybe `[I<options> ...]`.

Another example case is in the `-macopt` section.

```
B<-macopt> I<nm>B<:>I<v> ...
```

There, I chose to leave it unitalicized because the repeatable element it's referring to is a mix of `I<>` and `B<>` stuff, so I picked the more generic plain text that indicates a general "syntactic element". Since we don't have a formal grammar and the `I<nm>B<:>I<v>` isn't surrounded by grouping elements, it might be ambiguous what the repetition refers to, but it seems readable. There may be better guidance in those `mdoc` man pages that kaduk pointed me at; I haven't gotten to read those yet.

I'll try de-italicizing all the '...'s. That would make it more consistent, and (if I understand it) richsalz's reasoning about it being a syntactic element (as opposed to a placeholder token, right?) makes sense.